packages/api/src/routes/medical/internal-facility.ts

```diff
@@ -50,9 +50,23 @@ router.put(
       },
     };
 
-    const facility = await registerFacilityWithinHIEs(cxId, facilityUpdate);
+    const f = await registerFacilityWithinHIEs(cxId, facilityUpdate);
 
-    return res.status(httpStatus.OK).json(facility);
+    return res.status(httpStatus.OK).json({
+      id: f.id,
+      etag: f.eTag,
+      name: f.data.name,
+      npi: f.data.npi,
+      tin: f.data.tin,
+      active: f.data.active,
+      address: f.data.address,
+      cqType: f.cqType,
+      cqActive: f.cqActive,
+      cqOboOid: f.cqOboOid,
+      cwType: f.cwType,
+      cwActive: f.cwActive,
+      cwOboOid: f.cwOboOid,
+    });
```

Comment:
I suggest we have a `toDto()` or something, so we can have standard response types across diff endpoints.

If you prefer to ship as is and improve this on a follow-up PR, that's fine - I'd just ask that we add it to the master ticket and address before it gets out of hand and become too large of a lift that justifies the refactoring.